package/autoyast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jul  4 14:51:50 UTC 2017 - jlopez@suse.com
+
+- storage-ng: create a new guided proposal by changing settings if
+  it is necessary
+- 3.3.4
```

Comment:
Do we really need to increase the version number? We are submitting nowhere and nothing depends on this. So I don't think so.

But if you increase version, please jump to 3.4.x, master is already 3.3.x and we want to stay ahead.